modules/admin_manual/pages/_partials/configuration/server/occ_command/system_command.adoc

```diff
@@ -7,11 +7,23 @@ To execute xref:configuration/server/background_jobs_configuration.adoc[backgrou
 sudo -u www-data php occ system:cron
 ....
 
-[IMPORTANT]
-====
-The `system:cron` command replaces https://doc.owncloud.com/server/admin_manual/configuration/server/background_jobs_configuration.html#cron[System cron], which used `</path/to/your/owncloud>/cron.php`.
-cron.php will continue to work with ownCloud Server 10.3, but will be removed in a later version, so you are strongly encouraged to begin using `system:cron`.
-====
+== Integrating with System Cron
+
+If you’re automating background jobs via Cron (or plan to), update the relevant `crontab` entry, using the example below as a guide. 
+
+[source,console]
+----
+# Instead of the following configuration
+/usr/bin/php -f /path/to/your/owncloud/cron.php`
+
+# Use the following one instead
+/usr/bin/php -f /path/to/your/owncloud/occ system:cron` 
+----
+
+== Fallback Support
```

Comment:
Please remove that fallback section. As mentioned in the release notes this is just a graceful way of changing the behavior to really be sure that background jobs will continue to be executed. `cron.php` will be removed in a later version and nobody should use it anymore with > 10.3.

Reply:
Shall do.